### PR TITLE
Finalise localdispatch handling

### PR DESCRIPTION
**Pull request by sloops77**

### CODE REVIEW

File: packages/abstract-transport-queue/src/factory.js [hunk 1]
```diff
@@ -6,9 +6,12 @@ import QueueTransport from './QueueTransport';
 
 export default function configure({ queueSystem, queueSystemFactory }) {
   return function (options) {
-    const { source, log, requestTimeout, optimizeLocalHandlers } = options;
+    const { source, log, requestTimeout, optimizeLocalHandlers, standardiseDates } = options;
```
Comment (ozsay) [inline packages/abstract-transport-queue/src/factory.js]: is it the right term? `standardiseDates` seems closer to `serializableDates`

File: packages/abstract-transport-queue/src/factory.js [hunk 2]
```diff
@@ -6,9 +6,12 @@ import QueueTransport from './QueueTransport';
...
     const finalQueue = queueSystem || queueSystemFactory(assign({ log }, options));
-    const transport = new QueueTransport(finalQueue, source, log, requestTimeout);
-    return optimizeLocalHandlers ? localRequestHandlersDecorator(transport) : transport;
+    const TransportClazz = optimizeLocalHandlers
```
Comment (ozsay) [inline packages/abstract-transport-queue/src/factory.js]: rename file to `factory.java`

File: packages/core/package.json [hunk 3]
```diff
@@ -54,6 +54,7 @@
     "usePathForSuiteName": "true"
   },
   "devDependencies": {
+    "@stellarjs/abstract-transport": "^0.10.1",
```
Comment (ozsay) [inline packages/core/package.json]: no circular deps here?

Comment (sloops77) [reply packages/core/package.json]: only needed for tests...acceptable or not?

Comment (sloops77) [reply packages/core/package.json]: only needed for tests...acceptable or not?

Comment (ozsay) [reply packages/core/package.json]: i think you get a lerna warning, and not sure if it does anything bad

Comment (sloops77) [reply packages/core/package.json]: nah nothing bad -approve me pls

File: packages/core/src/StellarRequest.js [hunk 4]
```diff
@@ -52,22 +58,30 @@ export default class StellarRequest extends StellarCore {
...
-    this.fireAndForgetMiddlewares = this.handlerChain.concat(
-      {
-        fn(request) {
-          return me.transport.fireAndForget(request);
-        },
-      });
+    function buildFinalMws(dispatchFn) {
+      if (me.optimizeLocalHandlers) {
+        return [
+          { fn: mwLocalDispatchFactory({ stringifyDates: me.stringifyDates }) },
```
Comment (ozsay) [inline packages/core/src/StellarRequest.js]: i guess it's fine for now, but it's too much trouble.. we can put it in `config-stellar.js` in the project

Comment (sloops77) [reply packages/core/src/StellarRequest.js]: has to be the second last one. only way i can guarantee that is by putting it here

**Pull request merged**